### PR TITLE
ENG-3070 Disable workflows list sorting

### PR DESCRIPTION
**Pull request by eunice-chan**
## Describe your changes and why you are making these changes
Delay the workflows list rendering so that the dates can load in and the rows can order before loading. Will be replaced with a more ideal solution as a future task. Not sure if having a several seconds delay before loading the workflows page is preferred to showing the reordering.

Tested with 100 workflows (modified from Hari's many small workflows example but instead these workflows run once and aren't not scheduled) and S3 as metadata store.

## Related issue number (if any)
ENG-3070
ENG-3096

## Loom demo (if any)
Jun 5 Update:
No ordering:

https://github.com/aqueducthq/aqueduct/assets/30596854/7a7eabc4-218a-4b17-a222-17307a297c32

---
Before

https://github.com/aqueducthq/aqueduct/assets/30596854/f0d5266a-c9b5-43b3-a9b2-933be867da89

After

https://github.com/aqueducthq/aqueduct/assets/30596854/2e0a5011-172f-43a1-acdb-f056d65d4088

## Checklist before requesting a review
- [x] I have created a descriptive PR title. The PR title should complete the sentence "This PR...".
- [x] I have performed a self-review of my code.
- [x] I have included a small demo of the changes. For the UI, this would be a screenshot or a Loom video.
- [N/A] If this is a new feature, I have added unit tests and integration tests.
- [N/A] I have run the integration tests locally and they are passing.
- [x] I have run the linter script locally (See `python3 scripts/run_linters.py -h` for usage).
- [x] All features on the UI continue to work correctly.
- [x] Added one of the following CI labels:
    - `run_integration_test`: Runs integration tests
    - `skip_integration_test`: Skips integration tests (Should be used when changes are ONLY documentation/UI)

### CODE REVIEW

**Comment by eunice-chan** [comment]
No ordering:
https://github.com/aqueducthq/aqueduct/assets/30596854/7a7eabc4-218a-4b17-a222-17307a297c32

File: src/ui/common/src/components/layouts/NavBar.tsx [hunk 1]
```diff
@@ -118,7 +117,7 @@ const NavBar: React.FC<{
               );
             }
             return (
-              <Link
+              <RouterLink
```
Comment (likawind) [inline src/ui/common/src/components/layouts/NavBar.tsx:120]: Did this work? Curious if this helps disabling the refresh when you click on the breadcrumb link

File: src/ui/common/src/components/pages/workflows/index.tsx [hunk 2]
```diff
@@ -308,10 +305,11 @@ const WorkflowsPage: React.FC<Props> = ({ user, Layout = DefaultLayout }) => {
           onChangeRowsPerPage={onChangeRowsPerPage}
           savedRowsPerPage={getRowsPerPage()}
           sortColumns={sortColumns}
-          defaultSortConfig={{
-            sortColumn: sortColumns[0],
-            sortType: SortType.Descending,
-          }}
+          // Disabled until ENG-3096
```
Comment (likawind) [inline src/ui/common/src/components/pages/workflows/index.tsx:308]: Nit: Add a `TODO:` in this comment. This helps us finding and clean up `TODO`s if these follow up didn't get in due to other priorities.

File: src/ui/common/src/components/layouts/NavBar.tsx (hunk 1 above)
Comment (eunice-chan) [reply src/ui/common/src/components/layouts/NavBar.tsx:120]: This is how it looks:

https://github.com/aqueducthq/aqueduct/assets/30596854/e37c5036-4426-4b50-a2b3-bc62db672bb2

I wasn't too sure what you meant by disabling the refresh.

Comment (likawind) [reply src/ui/common/src/components/layouts/NavBar.tsx:120]: Oh I meant did you see lots of API call in your server log when you click on the link? Previously it does as we refetch these APIs, which didn't utilize the caching power of RTK framework

**Review by likawind — APPROVED** [review]
LGTM, thanks for following up and addressing multiple rounds of comments!

File: src/ui/common/src/components/layouts/NavBar.tsx (hunk 1 above)
Comment (likawind) [reply src/ui/common/src/components/layouts/NavBar.tsx:120]: It's not a big deal if it doesn't fix. We can file a task and investigate later

Comment (eunice-chan) [reply src/ui/common/src/components/layouts/NavBar.tsx:120]: I think it does. When clicking "workflows" from the workflow detail page of workflow 0, only the details of the workflow 0 was filled in. The rest had to be loaded in from the server.

Comment (likawind) [reply src/ui/common/src/components/layouts/NavBar.tsx:120]: Hmm okay let's file a separate task then

**Pull request merged**